AppCenterDistribute/AppCenterDistributeTests/MSDistributeCheckForUpdateTests.m

```diff
@@ -200,8 +200,8 @@ - (void)testCheckForUpdateGetsLatestReleaseEvenThoughAutomaticUpdateIsDisabled {
 
   // Ingestion mock.
   __block id ingestionMock = OCMClassMock([MSDistributeIngestion class]);
+  __block void (^handler)(NSString *callId, NSHTTPURLResponse *_Nullable response, NSData *_Nullable data, NSError *_Nullable error);
   OCMStub([ingestionMock checkForPublicUpdateWithQueryStrings:OCMOCK_ANY completionHandler:OCMOCK_ANY]).andDo(^(NSInvocation *invocation) {
-    void (^handler)(NSString *callId, NSHTTPURLResponse *_Nullable response, NSData *_Nullable data, NSError *_Nullable error);
```

Comment:
Why is it moved outside? if we really need it, `retainArguments` should be called. But I don't see the problems with previous version of code

Reply:
Good catch. It was for testing and wasn't reverted. Reverted in #2004.

Reply:
I got confused with others. All others were reverted after test except this one because of a crash when it releases objects. Let's move the discussion to the new PR.